kuadrant-dev-setup/rbac/rhdh-rbac.yaml

```diff
@@ -10,39 +10,39 @@ kind: ClusterRole
 metadata:
   name: rhdh-kuadrant-reader
 rules:
-- apiGroups: ["kuadrant.io"]
-  resources:
-    - authpolicies
-    - ratelimitpolicies
-    - dnspolicies
-    - tlspolicies
-  verbs: ["get", "list", "watch"]
-- apiGroups: ["extensions.kuadrant.io"]
-  resources:
-    - planpolicies
-    - apiproducts
-  verbs: ["get", "list", "watch"]
-- apiGroups: ["extensions.kuadrant.io"]
-  resources:
-    - apikeyrequests
-  verbs: ["get", "list", "watch", "create", "delete", "patch", "update"]
-- apiGroups: ["extensions.kuadrant.io"]
-  resources:
-    - apikeyrequests/status
-  verbs: ["get", "patch", "update"]
-- apiGroups: ["gateway.networking.k8s.io"]
-  resources:
-    - gateways
-    - httproutes
-  verbs: ["get", "list", "watch"]
-- apiGroups: [""]
-  resources:
-    - namespaces
-  verbs: ["get", "list", "watch"]
-- apiGroups: [""]
-  resources:
-    - secrets
-  verbs: ["get", "list", "watch", "create", "delete"]
+  - apiGroups: ["kuadrant.io"]
+    resources:
+      - authpolicies
+      - ratelimitpolicies
+      - dnspolicies
+      - tlspolicies
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["extensions.kuadrant.io"]
+    resources:
+      - planpolicies
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["extensions.kuadrant.io"]
+    resources:
+      - apiproducts
```

Comment:
This is the relevant change: apiproduct resource now has granted permission for all the verbs. For the rest of the changes, blame the LSP of my IDE.